schemas/automated-liquidity-management/schema.json

```diff
@@ -0,0 +1,253 @@
+{
+  "schema": "Automated Liquidity Management",
+  "description": "Standard table definitions for automated liquidity management protocol.",
+  "protocolCategory": ["Automated Liquidity Management"],
+  "version": "1.0.0-alpha",
+  "tables": [
+    {
+      "label": "Pools",
+      "tableName": "automated_liquidity_pools",
+      "aggregation": "none",
+      "uniqueKey": ["pool_address"],
+      "description": "List of pools in the protocol.",
+      "properties": {
+        "chain_id": {
+          "description": "The standard chain id.",
+          "type": "number"
+        },
+        "timestamp": {
+          "description": "The timestamp this pool was created.",
+          "type": "number"
+        },
+        "creation_block_number": {
+          "description": "The block number that this pool was created.",
+          "type": "number"
+        },
+        "strategy_vault_contract_address": {
+          "description": "The contract address of the strategy vault which manages the liquidity pool positions.",
+          "type": "string"
+        },
+        "underlying_liquidity_pool_address": {
+          "description": "The contract address of the underlying liquidity pool where liquidity are deposited into",
+          "type": "string"
+        },
+        "strategy_vault_receipt_token_address": {
+          "description": "The contract address of ERC20 token which represents the share of liquidity provided.",
+          "type": "string"
+        },
+        "strategy_vault_receipt_token_decimals": {
+          "description": "The decimal amount of the ERC20 receipt token.",
+          "type": "number"
+        },
+        "strategy_vault_receipt_token_symbol": {
+          "description": "The symbol of the receipt token.",
+          "type": "string"
+        }
+      }
+    },
+    {
+      "label": "Position Snapshot",
+      "tableName": "automated_liquidity_depositors",
+      "aggregation": "daily",
+      "uniqueKey": ["user_address", "pool_address", "block_date"],
+      "description": "Snapshot of the pool users.",
+      "properties": {
+        "timestamp": {
+          "description": "The timestamp of the snapshot.",
+          "type": "number"
+        },
+        "block_date": {
+          "description": "The timestamp truncated (ie, YYYY-MM-DD format for daily snapshots and YYYY-MM-DD HH:00:00 for hourly snapshots).",
+          "type": "date"
+        },
+        "chain_id": {
+          "description": "The standard chain id.",
+          "type": "number"
+        },
+        "strategy_vault_contract_address": {
+          "description": "The address of the strategy vault this user has a position in.",
+          "type": "string"
+        },
+        "user_address": {
+          "description": "The address of the user who has a position in the strategy vault.",
+          "type": "string"
+        },
+        "underlying_liquidity_pool_address": {
```

Comment:
Can we call this `liquidity_pool_address` or `pool_address` same with the pools table? Underlying seems redundant to say, isn't there only one liquidity pool address?